src/transformers/generation_utils.py

```diff
@@ -691,6 +693,8 @@ def _get_logits_processor(
         exponential_decay_length_penalty: Tuple,
         logits_processor: Optional[LogitsProcessorList],
         renormalize_logits: Optional[bool],
+        supress_tokens: Optional[List[int]],
```

Comment:
```suggestion
        suppress_tokens: Optional[List[int]],
```
Suppress is with double p :)

Reply:
Oups thanks